spec/collator.html

```diff
@@ -78,6 +78,10 @@ <h1>InitializeCollator ( _collator_, _locales_, _options_ )</h1>
         1. Return _collator_.
       </emu-alg>
     </emu-clause>
+
+    <emu-note>
+      The collation activated by the `search` value for the `usage` option should only be used to find matching strings, since this collation is not guaranteed to be in any particular order.
```

Comment:
```suggestion
      The collation associated with the *"search"* value for the *"usage"* option should only be used to find matching strings, since this collation is not guaranteed to be in any particular order (as described in [Unicode Technical Standard #35 Part 1 Core, Unicode Collation Identifier](https://unicode.org/reports/tr35/#UnicodeCollationIdentifier) and [Unicode Technical Standard #10 Unicode Collation Algorithm, Searching and Matching](https://unicode.org/reports/tr10/#Searching)).
```